channel.go

```diff
@@ -0,0 +1,340 @@
+package cli
+
+import (
+	"encoding/json"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/cheynewallace/tabby"
+	"github.com/urfave/cli/v2"
+)
+
+func NewChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "channel",
+		Usage:       "Manage channels.",
+		Description: "Manage channels.",
+		Subcommands: []*cli.Command{
+			getChannelCmd(config),
+			newListChannelsCmd(config),
+			newCreateChannelCmd(config),
+			newDeleteChannelCmd(config),
+			newUpdateChannelCmd(config),
+		},
+	}
+}
+
+func getChannelCmd(config *Config) *cli.Command {
+	return &cli.Command{
+		Name:        "get",
+		Usage:       "Get a channel by channel type and channel name.",
+		Description: "Get a channel by channel type and channel name.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "channel-type",
+				Aliases:  []string{"t"},
+				Usage:    "The type of the channel. Such as 'messaging' or 'livestream'.",
+				Required: true,
+			},
+			&cli.StringFlag{
+				Name:     "channel-name",
+				Aliases:  []string{"n"},
+				Usage:    "The name of the channel.",
+				Required: true,
+			},
+			&cli.BoolFlag{
+				Name:     "json",
```

Comment:
What would be the need of this?

Reply:
I don't know, I coped it from the Node CLI. Ferhat said that it'd be nice to have a somewhat similar feature set to the old CLI tool.

Reply:
Yeah but if there is no explicit need i think we can get rid of it. Honestly i don't see any value of this.

Reply:
The value is that we only print just certain fields. The --json flags makes you able to read every field.

Additionally, you can parse it yourself and use it in scripts. See here: https://github.com/GetStream/stream-cli#-fun-facts